libs/core/mappers/anime.mapper.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+
+import { AnimeDto } from '../dtos/anime.dto';
+import { Anime } from '../models/anime.model';
+import { PaginationDto } from '../dtos/pagination.dto';
+import { AnimePagination } from '../models/anime-pagination.model';
+
+import { ANIME_TYPE_FROM_DTO } from './anime-type.mapper';
+import { ANIME_STATUS_FROM_DTO } from './anime-status.mapper';
+
+/** Service for mapping data. */
+@Injectable({
+	providedIn: 'root',
+})
+export class AnimeMapper {
+	/**
+	 * Method from Dto to model.
+	 * @param dto Data from API.
+	 */
+	public fromDto(dto: PaginationDto<AnimeDto>): AnimePagination<Anime> {
+		return {
+			count: dto.count,
+			next: dto.next,
+			previous: dto.previous,
+			results: dto.results.map(
+				(result: AnimeDto): Anime => ({
```

Comment:
You can omit such types, they should be fully inferable. 
```suggestion
				result => ({
```